src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -562,4 +554,8 @@ public void setEpilog(String epilog) {
     public String getEpilog() {
         return epilog;
     }
+
+    public EventBus getEventBus() {
```

Comment:
For me the Eventbus is an implementation-specific thing and so far nobody outside of the database class has to know about it. Thus hide it. If it is really needed, then the getEventBus can be readded later.